src/alchemlyb/tests/test_units.py

```diff
@@ -0,0 +1,40 @@
+import pytest
+import pandas as pd
+
+import alchemlyb
+from alchemtest.gmx import load_benzene
+from alchemlyb.parsing.gmx import extract_dHdl
+from alchemlyb.postprocessors.units import to_kT
+
```

Comment:
Add an XFAIL test for using `pandas.concat`. When this test passes we know that pandas fixed attrs.

Reply:
Add an XFAIL test for using pandas loc/iloc slicing. When this test passes we know that pandas fixed attrs.

Reply:
@orbeckst loc/iloc slicing is safe now with pandas>=1.2